tests/rptest/scale_tests/tiered_storage_single_partition_test.py

```diff
@@ -237,7 +238,13 @@ def metadata_readable():
         self.logger.info(
             f"Delta: {offset_delta}, Segments: {segment_count}, Last term {last_term}"
         )
-        assert offset_delta <= (2 * segment_count + last_term)
+
+        # If we upload in batches of four segments at a time
+        min_offset_delta = (segment_count * 1.5) + last_term
```

Comment:
Nice, just making sure, these are (1 + 1/4 + 1/4) and (1 + 1/1 + 1/1) respectively, the 1/4 corresponding to the best case where 4 segments are uploaded at once and the manifest is marked clean for those 4

Reply:
yes, the min is 1 per segment + (1 + 1 every four segments), and the max is (1 + 1 + 1) per segment.